Aptoide/aptoide-client-v8
# [bugfix/ASV-1398] Reversing TargetSdkVersion back to 25

**What does this PR do?**

   This basically reverses [this PR.](https://github.com/Aptoide/aptoide-client-v8/pull/852) because of the issue that is currently not handled described [here](https://developer.android.com/guide/components/broadcast-exceptions).

**Database changed?**

   No

**How should this be manually tested?**

  Check if everything is back to normal, especially check if after installing, the the "Install" button on AppView is replaced by the "Open" button.

**What are the relevant tickets?**

  [ASV-1398](https://aptoide.atlassian.net/browse/ASV-1398)



**Code Review Checklist**
